src/__tests__/getLocationForJsonPath.spec.ts

```diff
@@ -76,6 +78,116 @@ describe('getLocationForJsonPath', () => {
     });
   });
 
+  describe('merge keys fixture', () => {
+    const result = parseWithPointers(mergeKeys, { mergeKeys: true });
```

Comment:
Maybe let's change the variable holding the content — I see twice `mergeKeys` and it's confusing.